Sources/Views/ALKInformationCell.swift

```diff
@@ -76,31 +84,110 @@ final class ALKInformationCell: UITableViewCell {
 
     func update(viewModel: ALKMessageViewModel) {
         self.viewModel = viewModel
+        
+        guard let dictionary = getFeedback(viewModel: viewModel) else { return }
+        var comment = ""
+        var rating = 0
+        
+        if dictionary["comments"] != nil {
+            contentView.subviews.forEach { subview in
+                subview.removeFromSuperview()
+            }
+            comment = dictionary["comments"]! as! String
+        }
 
-        messageView.text = viewModel.message
+        if dictionary["rating"] != nil {
+            contentView.subviews.forEach { subview in
+                subview.removeFromSuperview()
+            }
+            rating = dictionary["rating"]! as! Int
+        }
+        
+        let imageAttachment = NSTextAttachment()
+        imageAttachment.image = RatingHelper().getRatingIconFor(rating: rating)
+        guard let attachedImage = imageAttachment.image else { return }
+        imageAttachment.bounds = CGRect(x: 0, y: -5 , width: attachedImage.size.width, height: attachedImage.size.height)
+        let imageString = NSAttributedString(attachment: imageAttachment)
+        let userLabel = localizedString(forKey: "RatingLabelTitle", withDefaultValue: SystemMessage.Feedback.RatingLabelTitle, fileName: configuration.localizedStringFileName)
+        let textString = NSMutableAttributedString(string: userLabel + " " + viewModel.message! + "  ")
+        textString.append(imageString)
+        messageView.attributedText = textString
+        if !comment.isEmpty {
+            commentTextView.text = "“\(comment)”"
+        }else{
+            commentTextView.text = ""
+        }
+        
+        setUpConstraintsForRating()
+        setupStyle()
```

Comment:
This we may have to call even the feedback is not present.

Reply:
done